README.md

```diff
@@ -1,59 +1,78 @@
 # Liberty Config Language Server
 
-Provides language server features for Liberty server configuration files. Supported files include 
-- `server.env`
-- `bootstrap.properties`
-- `server.xml` (or any XML file) and its referenced XML files through the `<include>` element
-    - XML file must contain the root element `<server>` and exist in `src/main/liberty/config`, `configDropins/overrides`, `configDropins/defaults`, `usr/shared/config`, or `usr/servers` directory
+The Liberty Config Language Server provides language server features for Liberty server configuration files through any of the supported client IDEs. It adheres to the [language server protocol](https://github.com/Microsoft/language-server-protocol).
+
+## Client IDEs
 
-Liberty Config Language Server adheres to the [language server protocol](https://github.com/Microsoft/language-server-protocol)
-and is available for use with the following clients.
+The Liberty Config Language Server is included in the following client IDEs.
 
-# Client IDEs
 * [Liberty Tools for VS Code](https://github.com/OpenLiberty/liberty-tools-vscode)
 * [Liberty Tools for Eclipse](https://github.com/OpenLiberty/liberty-tools-eclipse)
 * [Liberty Tools for Intellij](https://github.com/OpenLiberty/liberty-tools-intellij)
 
+## Supported files
+
+Liberty Config Language Server features are avaialble for the following configuration files.
+
+- `server.env`
+- `bootstrap.properties`
+- `server.xml` (or any XML file) and any XML files that are referenced through the `include` element in the `server.xml` file.
```

Comment:
this bullet is a little confusing. I'm not sure what to make of "or any XML file", esp since we set limitations both in this sentence and in the sub bullet. What do we want to let users know by saying "any XML file" ?

Also

> XML files must contain the `server` root element and exist in the...

Does this also apply to included files?

Reply:
Basically, we recognize `server.xml` files, but also xml files by other names when they meet the criteria that followed (located in a certain directory and have a `server` root element).

Reply:
do files that are referenced through the `include` element in the `server.xml` file also need to meet the root element and file location criteria?

Reply:
They need the root `server` element, but they can be located anywhere.